README.adoc

```diff
@@ -1,53 +1,72 @@
 
 [cols="1,1,1",options="autowidth,header"]
 |===
 |Current Chart Version |Original Repository |Default Values
-|*{chart-version}* |{original-repo-url}/tree/main/charts/velero[Chart] | https://artifacthub.io/packages/helm/vmware-tanzu/velero/4.1.3?modal=values[`values.yaml`]
+|*{velero-chart-version}* |{original-repo-url}/tree/main/charts/velero[Chart] | https://artifacthub.io/packages/helm/vmware-tanzu/velero/{velero-chart-version}?modal=values[`values.yaml`]
 |===
 
 == Usage
 
-_Here you will show examples of how to call this module on your code. Note that the `depends_on` needs to be adapted to your use case._
+=== Backup
+
+Here is an example of the module usage:
 
-_You can start by giving a simple example and then show some others with increasing complexity and/or for specific use cases._
+- A default backend in a AWS S3 bucket, accessed with a ServiceAccount
+- A schedule at 4 a.m, which uses EBS snapshots as backend for PVC
+- A schedule at 2 a.m, which uses Restic as backend for PVC
+- Prometheus monitoring with a Grafana dashboard
 
 [source,terraform]
 ----
 module "backup" {
-  # source           = "git::https://github.com/camptocamp/devops-stack-module-backup.git//eks?ref=grafana"
-  source           = "../module-backup/eks"
-  target_revision  = "grafana"
+  source           = "git::https://github.com/camptocamp/devops-stack-module-velero.git//eks"
   cluster_name     = local.cluster_name
   base_domain      = local.base_domain
   argocd_namespace = module.argocd_bootstrap.argocd_namespace
 
+  ``
   enable_monitoring_dashboard = true
 
+  # Automated backups can be specified.
+  #
+  # The included Resources is very important: if the resource `pods` is specified,
+  # then velero will not attempt to use the native snapshot feature of the object
+  # storage of the cloud provider (EBS snapshots), and will use the File System Backup, i.e
+  # Restic.
   backup_schedules = {
-    snapshot-backup = {
+
+    # This schedule will use the native snapshot ability to backup the PVC
+    snapshot-schedule = {
+      # Can temporarly disable snapshot schedules
       disabled = false
-      schedule = "1 4 * * *"
+      # Cron-style scheduling
+      schedule = "2 4 * * *"
       template = {
-        # storageLocation    = "backup-bucket"
+        # Select the backend for this schedule
+        storageLocation    = "backup-bucket"
         includedNamespaces = ["wordpress"]
         includedResources  = ["persistentVolumes", "persistentVolumeClaims"]
       }
     },
-    restic-backup = {
+    # This schedule will force the Restic backend to backup the PVC
+    restic-schedule = {
       disabled = false
-      schedule = "5 4 * * *"
+      schedule = "2 2 * * *"
       template = {
-        # storageLocation    = "backup-bucket"
-        includedNamespaces = ["wordpress", "velero"]
+        # If using an non-default location:
```

Comment:
It's not clear to me what this means. Maybe it's just me not being familiarized with the module behaviour.